src/BlockEntities/BrewingstandEntity.h

```diff
@@ -119,6 +126,9 @@ class cBrewingstandEntity :
 	/** Amount of ticks that the current item has been brewed */
 	short m_TimeBrewed;
 
+	/** The fuel for the brewing stand */
+	short m_Fuel;
```

Comment:
The fuel means how many brewing operations can be done.
I would say new name `FuelRemaining` and the doxy comment, to:
`Amount of brewing operations that can be done with the remaining fuel.`